ImageKitSample/Program.cs

```diff
@@ -0,0 +1,426 @@
+
+            // GetFileVersionDetails
+            ResultFileVersionDetails resultFileVersionDetails = imagekit.GetFileVersionDetails("file_Id", "version_Id");
+
+            #endregion
+
+            #region ManageFolder
+
+            CreateFolderRequest createFolderRequest = new CreateFolderRequest
+            {
+                FolderName = "folder_name",
+                ParentFolderPath = "source/folder/path",
+            };
+            ResultEmptyBlock resultEmptyBlock = imagekit.CreateFolder(createFolderRequest);
+
+            // DeleteFolderRequest
+            DeleteFolderRequest deleteFolderRequest = new DeleteFolderRequest
+            {
+                FolderPath = "source/folder/path/folder_name",
+            };
+            ResultNoContent resultNoContent2 = imagekit.DeleteFolder(deleteFolderRequest);
+
+            // CopyFolder
+            CopyFolderRequest cpyFolderRequest = new CopyFolderRequest
+            {
+                SourceFolderPath = "path_1",
+                DestinationPath = "path_2",
+                IncludeFileVersions = true,
+            };
+
+            ResultOfFolderActions resultOfFolderActions = imagekit.CopyFolder(cpyFolderRequest);
+
+            // MoveFolder
+            MoveFolderRequest moveFolderRequest = new MoveFolderRequest
+            {
+                SourceFolderPath = "path_1",
+                DestinationPath = "path_2",
+            };
+
+            ResultOfFolderActions resultOfFolderActions1 = imagekit.MoveFolder(moveFolderRequest);
+
+            #endregion
+
+            #region GetBulkJobStatus
+
+            ResultBulkJobStatus resultBulkJobStatus = imagekit.GetBulkJobStatus("job_Id");
+
+            #endregion
+
+            #region  Purge
+
+            ResultCache resultCache = imagekit.PurgeCache("url");
+
+            ResultCacheStatus resultCacheStatus = imagekit.PurgeStatus("request_Id");
+
+            #endregion
+
+            #region  Metadata
+            ResultMetaData resultMetaData = imagekit.GetFileMetadata("file_Id");
+
+            ResultMetaData resultMetaData1 = imagekit.GetRemoteFileMetadata("https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg");
+
+            // CustomMetaDataFields
+            ResultCustomMetaDataFieldList resultCustomMetaDataFieldList = imagekit.GetCustomMetaDataFields(true);
+
+            // CreateCustomMetaDataFields
+            CustomMetaDataFieldCreateRequest requestModel = new CustomMetaDataFieldCreateRequest
+            {
+                Name = "field_name",
+                Label = "field_lable",
+            };
+            CustomMetaDataFieldSchemaObject schema = new CustomMetaDataFieldSchemaObject
+            {
+                Type = CustomMetaDataFieldSchemaObject.CustomMetaDataTypeEnum.Number,
+                MinValue = 1000,
+                MaxValue = 3000,
+                MinLength = 500,
+                MaxLength = 600,
```

Comment:
This example as it is, won't work. Because in case of Number type `MinLength` would not work. So if you want to include these options as well, maybe we can comment it out.